src/vif.jl

```diff
@@ -0,0 +1,119 @@
+# putting this behind a function barrier means that the model matrix
+# can potentially be freed immediately if it's large and constructed on the fly
+function _find_intercept(model::RegressionModel)
+    modelmat = modelmatrix(model)
+    # XXX collect is necessary for Julia 1.6
+    # but it's :just: an array of references to views, so shouldn't be too
+    # expensive
+    return findfirst(Base.Fix1(all, isone), collect(eachcol(modelmat)))
```

Comment:
How about using a `VERSION` check? That way we won't forget dropping it when we drop Julia 1.6 support and we don't waste memory on newer Julia versions.

Reply:
done